tools/distpartitioning/utils.py

```diff
@@ -64,37 +95,26 @@ def get_node_types(schema):
         dictionary with ntype <-> type_nid mappings
         list of ntype strings
     """
```

Comment:
the docstring for the return values is out of date.

Reply:
Updated accordingly.